packages/picasso/src/Section/Section.tsx

```diff
@@ -150,7 +150,13 @@ export const Section = forwardRef<HTMLDivElement, Props>(function Section(
         </Container>
       )}
       <Collapse in={!collapsed} unmountOnExit>
-        {children}
+        <Container
+          className={cx({
+            [classes[`${variant}SectionContent`]]: true,
+          })}
```

Comment:
Why not just use this?
```suggestion
          className={classes[`${variant}SectionContent`]}
```